Userland/Libraries/LibGfx/Filters/StackBlurFilter.cpp

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) 2010, Mario Klingemann <mario@quasimondo.com>
+ * Copyright (c) 2022, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#if defined(__GNUC__) && !defined(__clang__)
+#    pragma GCC optimize("O3")
+#endif
+
+#include <AK/Array.h>
+#include <AK/Vector.h>
+#include <LibGfx/Filters/StackBlurFilter.h>
+
+namespace Gfx {
+
+constexpr Array<u16, 255> mult_table {
```

Comment:
Can we get an equation for these tables?

Reply:
I can probably try to derive it later (and maybe replace these with some `constexpr` functions). The tables anyway are used to approximate a division by radius^2